docs/apply.md

```diff
@@ -0,0 +1,61 @@
+# How to apply
+
+1.  Read this reference guide, which will help you to apply and perform the different tests needed to evaluate your
+    solution.
+2.  If it is needed, set up the validation environment.
+3.  Execute tests according to the test workflow using the appropriate public instances or using your own deployed
+    instance.
+4.  Fill in the
+    [**FIWARE-ready** IoT Device Manual Template](https://s3.amazonaws.com/fundingbox-sites/gear%2F1517994518659-FIWARE+Ready+IoT+Device+Manual+Template+v1+%282%29.docx)
+    with the information about how to use your component within FIWARE ecosystem.
+5.  Go to `http://marketplace.fiware.org/apply`. Register, and then apply by providing all your company information.
+    Then, under the **solution/service** section, choose **FIWARE-ready IoT Device** under **category** and a proper
+    domain of application.
+6.  You might be asked to provide extra information or to have an interview with **FIWARE-ready** IoT Device reviewers
+    who might require extra information about your solution so that they can take an objective validation decision.
+7.  **FIWARE-ready** IoT Device reviewers will evaluate the documentation provided and will perform themselves the
```

Comment:
Ups... it means that the candidates have to provide the corresponding HW to do the tests. It is an important point because it is the different between a certification and accreditation process...

Reply:
fixed ddb2f84